src/C-DEngine/C-DBaseClasses/cde_BaseAssets.cs

```diff
@@ -52,7 +52,7 @@ public static string LoadCrypto(string pDLLName, ICDESystemLog pMySYSLOG = null,
                 Dictionary<string, string> tL = new();
                 Assembly tCryptoAssembly = null;
                 string codeSignThumb;
-                if (AppDomain.CurrentDomain?.FriendlyName != "RootDomain" && AppDomain.CurrentDomain?.FriendlyName != "MonoTouch") //Android and IOS
+                if (AppDomain.CurrentDomain?.FriendlyName != "RootDomain" && AppDomain.CurrentDomain?.FriendlyName != "MonoTouch" && AppDomain.CurrentDomain?.FriendlyName != "Meadow.dll") //Android and IOS
```

Comment:
the whole crypto will not work on Meadow. Dynamic loading of DLLs and dynamics are not supported yet. basically for Meadow crypto has to be turned off.

Reply:
In order to load a custom crypto the codesign of the crypto.lib has to be validated. Since this is not supported on these "sandboxed" platforms they have to be excluded here. I am not too happy about using the FriendlyName of the CurrentDomain but I have not found another way at this point to ignore these platforms from CodeSigning check. We need to revisit if this becomes an issue